nncf/common/quantization/quantizer_propagation/graph.py

```diff
@@ -1510,3 +1518,15 @@ def traverse_fn(curr_node_key: str, unused) -> Tuple[bool, Any]:
                     == QuantizerPropagationStateGraphNodeType.OPERATOR
                 )
                 assert pq in op_node[QuantizerPropagationStateGraph.AFFECTING_PROPAGATING_QUANTIZERS_ATTR]
+
+    def add_branching_quantizer_to_group(self, propagation_quantizer: PropagatingQuantizer, group_id: str) -> None:
+        """ "
+        Adds branching quantizer into the group associated by provided group ID.
+
+        :param propagation_quantizer: PropagatingQuantizer instance.
+        :param group_id: Group ID.
+        """
+        if group_id in self._partial_branch_merge_group_manager.get_group_vs_prop_quants_dict().keys():
```

Comment:
Looks like pylint missed it because couldn't assign dict type to function output
```suggestion
        if group_id in self._partial_branch_merge_group_manager.get_group_vs_prop_quants_dict():
```